.github/workflows/flake8.yml

```diff
@@ -1,5 +1,9 @@
 name: flake8
-on: [pull_request]
+on:
+  pull_request:
+    paths:
+      - 'library/**.py'
+      - 'tests/library/**.py'
```

Comment:
we also have `./tests/conftest.py` and everything in `./tests/functional/tests/`

Reply:
but we're not testing those files at the moment.

Should we wait for #5897 to be merged first ?

Reply:
> but we're not testing those files at the moment.

I wouldn't be concerned because of that anyway :man_shrugging:

Reply:
> Should we wait for #5897 to be merged first ?

Why would we wait for #5897 ? You can still edit this patch anyway